src/gui/guiHyperText.cpp

```diff
@@ -17,31 +17,26 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 */
 
-#include "IGUIEnvironment.h"
-#include "IGUIElement.h"
+#include "guiHyperText.h"
 #include "guiScrollBar.h"
-#include "IGUIFont.h"
-#include <vector>
-#include <list>
-#include <unordered_map>
-using namespace irr::gui;
 #include "client/fontengine.h"
-#include <SColor.h>
 #include "client/tile.h"
 #include "IVideoDriver.h"
```

Comment:
`IVideoDriver.h` is another Irrlicht header that can be removed.

Reply:
but it uses that?
![grafik](https://user-images.githubusercontent.com/1042418/148592581-9bda92ff-bcad-4be2-9b83-ca4fceb79285.png)

Reply:
It uses other Irrlicht headers that you removed, but Irrlicht headers are usually hidden under Minetest's `irrlichttypes[_[extra]bloated].h`.  In this case, Irrlicht's `IVideoDriver.h` is included by Minetest's `renderingengine.h`.  It really doesn't matter though; I was just recommending it for consistency.